frontend/src/app/features/in-app-notifications/center/menu/menu.component.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnInit,
+} from '@angular/core';
+import { combineLatest } from 'rxjs';
+import { map, tap } from 'rxjs/operators';
+import { StateService } from '@uirouter/core';
+import idFromLink from 'core-app/features/hal/helpers/id-from-link';
+import { I18nService } from 'core-app/core/i18n/i18n.service';
+import { INotificationPageQueryParameters } from '../../in-app-notifications.routes';
+import { IanMenuService } from './state/ian-menu.service';
+
+export const ianMenuSelector = 'op-ian-menu';
+
+@Component({
+  selector: ianMenuSelector,
+  templateUrl: './menu.component.html',
+  styleUrls: ['./menu.component.sass'],
+  providers: [ IanMenuService ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class IanMenuComponent implements OnInit {
+  baseMenuItems = [
+    {
+      title: 'Inbox',
+      icon: 'inbox',
+      href: this.getHrefForFilters({}),
+    },
+  ];
+
+  reasonMenuItems = [
+    {
+      key: 'mentioned',
+      title: '@mentioned',
+      icon: 'mention',
+      href: this.getHrefForFilters({ filter: 'reason', name: 'mentioned' }),
+    },
+    {
+      key: 'assigned',
+      title: 'Assigned',
+      icon: 'assigned',
+      href: this.getHrefForFilters({ filter: 'reason', name: 'assigned'  }),
+    },
+    {
+      key: 'accountable',
+      title: 'Accountable',
+      icon: 'accountable',
+      href: this.getHrefForFilters({ filter: 'reason', name: 'accountable'  }),
```

Comment:
The `accountable` filter does not work. I'm not sure what the correct alternative is. I've asked @ulferts but no answer yet.